.github/workflows/linux_hip.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   build:
     name: github/linux/hip/fast
     runs-on: ubuntu-latest
-    container: stellargroup/hip_build_env:10
+    container: pikaorg/pika-ci-hip:1
```

Comment:
Would it possible/easier to have HIP in the base image as well? Size is definitely one argument... Just curious what you think. I don't mind either way.

Reply:
The argument to have them separate is that hip is is quite long to install and we would need to do it everytime we build and push the image. But it's not a so strong argument since we don't update the image that often so I don't mind. Let me know what you prefer :)

Reply:
Fair enough. Let's keep them separate at the moment (the title does say "slimmer image" after all...).